open-policy-agent/opa
# Fix community page issues

### :nut_and_bolt: What code changed, and why?

This should fix some layout issues with the website community page, specifically the background should no longer turn white when scrolling, and there should no longer be a white column on the right of the page causing horizontal scrolling.

### :+1: Definition of done

- [x] The community page background stays put when scrolling
- [x] There's no white column to the right of the page, causing horizontal scrolling

### :athletic_shoe: How to test

1. Visit the [deploy preview](https://deploy-preview-4904--openpolicyagent.netlify.app/community)
2. Check that the background no longer displays a white background when scrolling
3. Check that the page doesn't scroll sideways anymore, and does not have a white column on the right hand side

### :chains: Related Resources

Signed-off-by: Marcus Stade [marcus@styra.com](mailto:marcus@styra.com)
